src/components/search/search-large.html

```diff
@@ -2,17 +2,17 @@
   <svg class="bx--search-magnifier">
     <use xlink:href="/carbon-icons/bluemix-icons.svg#icon--search--glyph"/>
   </svg>
-  <label class="bx--label" for="search__input">Search</label>
-  <input class="bx--search-input" type="text" id="search__input" placeholder="Search">
+  <label class="bx--label" id="search-input-label-2" for="search__input-1">Search</label>
+  <input class="bx--search-input" type="text" id="search__input-1" placeholder="Search" aria-labelledby="search-input-label-2">
```

Comment:
Since this search field is set to `type="text"` to remove the browser clear button, we should add `role="search"` to let users know this is a search input